src/crud/session.py

```
@@ -170,22 +203,24 @@ async def get_session(
+            .where(models.Session.name == session_name)
+        ),
     )
 
-    result = await db.execute(stmt)
-
-    honcho_session = result.scalar_one_or_none()
-
     if honcho_session is None:
         raise ResourceNotFoundException(
             f"Session {session_name} not found in workspace {workspace_name}"
         )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)